go/tools/macrobench/macrobench.go

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
this seems to be an empty file and can be removed.

Reply:
Definitely. Removed in 4c4a7d4fc9804e197b61707bfd76c0e6e2f4d517